generators/app/templates/_dockerTaskGeneric.sh

```diff
@@ -64,7 +69,7 @@ showUsage () {
     echo ""
     echo "Commands:"
     echo "    build: Builds a Docker image ('$imageName')."
-    echo "    compose: Runs docker-compose."
+    echo "    compose: Builds the images and runs docker-compose. Images are re-built when using release environment while debug environment uses a cached version of the image."
```

Comment:
This run-on needs some punctuation. Minimum add a comma after release environment.